### DISCUSSION TITLE
六分街日报

### DISCUSSION THREAD

**Discussion by yutong903** [share121/inter-knot#1828] [其他]
近日，六分街的录像店重新营业，街坊邻居对其表示由衷的祝贺，接下来让我们走进六分街，去探索这两位录像店店主的故事。我是本台记者，郁恸.克里斯提安娜。
![5c718c526fb33a83fbd6f5296dd7992f_6579831306703c7e2b4ddb89r7JisBWS05_fop=imageView%2F2%2Fw%2F600%2Ff%2Fjpeg%2Fq%2F60](https://github.com/user-attachments/assets/6bcf209b-82f3-4f9b-9d10-b2a620f43339)

今日的六分街如往日一般喧闹，让我们穿过穿过嘈杂的大街，去录像店里一探究竟吧
录像店店主之一，玲某：最近呢，我们的录像店终于重新开业了，在这之前，我们遇到了很多困难，在这里，感谢所有帮助过我们的人。
郁恸：方便说一下是什么样的困难吗？
玲某：可以的，其实......呃......就是那个人（指向另一位店主，哲某）那是我哥......他......肾虚，所以我们就到处求医，终于寻到了医生，尽力治好了他的肾......可是......可是......
郁恸：店主，怎么了？
玲某：可是他的病实在是太严重了，即使那位医生拼尽全力也无法治好......
![b4569f8dc26a52268faba9e15cac886b_2000x%3E](https://github.com/user-attachments/assets/fbeaf582-b58d-432e-9575-54881aefe5d0)
郁恸：真是令人......意想不到的坏消息
虽然身患重病，但仍然坚守工作岗位，为六分街的居民们带去笑容，这，就是那位店主的愿望，致敬传奇录像店店主，哲某。
好了，今日的日报就到这里，我们明天再见。

**Comment by gvfjh** [comment]
店主！没有你我可怎么活啊！😭

**Comment by biliqide** [comment]
可是，店长怎么会肾虚呢？🧐

**Reply by yutong903** [reply]
看好看的录像带看太多了😋

**Comment by Yitaruose-BaiXu** [comment]
求店长复活

**Comment by ZhuYuan-148** [comment]
不…会吧，前几天这个店长还很精神来着。
